lib/build.js

```diff
@@ -54,20 +55,21 @@ const newCleanup = (act, text) => {
             + key
             + " ".repeat(max_key_length - key.length)
             + " |-> "
-            + rewrites.join(" => ")
-          let comment_str = def && ("  // " + def.doc(key_) + "\n") || ""
+            + (isCreate ? rewrites[0] : rewrites.join(" => "))
+          let comment_str = def && def.doc(key_) && ("  // " + def.doc(key_) + "\n") || ""
           return comment_str + storage_str;
         })
         .join("\n")
-      return "storage" + (subjectName !== "ACCT_ID" ? " " + subjectName : "") + "\n" + storage + "\n";
+      let creates = isCreate ? "creates " : ""
+      return creates + "storage" + (subjectName !== "ACCT_ID" ? " " + subjectName : "") + "\n\n" + storage + "\n";
```

Comment:
How come we don't print `subjectName` if it's `== ACCT_ID`? Might be worth mentioning it in a comment.

Reply:
Commented :-)